### DISCUSSION TITLE
Secret Gists are Public

### DISCUSSION THREAD

**Discussion by altherlex** [community/community#39372] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

Secret gists are public. Am I missing anything?

**Comment by radar-blip** [comment, marked as answer]
Not missing anything,  they are not public.

![secretgist](https://user-images.githubusercontent.com/39196767/202581450-709b6080-fd5d-4865-9102-459a3b5200ac.png)

I searched for it and didn't see it, so what that tooltip says is correct.

**Reply by altherlex** [reply]
Thanks. I found an answer two minutes after publishing this issue and I couldn't delete this one.

Yes, secret gist doesn't mean private. They are accessible with a link.

Thanks again mate

**Reply by radar-blip** [reply]
You're welcome!

**Comment by shpark** [comment]
I learned this today. Until now, I thought secret gists can only be accessed by authenticated users 😂
